src/main.cc

```diff
@@ -70,6 +70,15 @@ static std::unique_ptr<Uptane::Target> find_target(LiteClient& client,
   bool find_latest = (version == "latest");
   std::unique_ptr<Uptane::Target> latest = nullptr;
   for (auto& t : client.allTargets()) {
+
+    if (!t.IsValid()) {
+      continue;
+    }
+
+    if (!t.IsOstree()) {
+      continue;
+    }
```

Comment:
this is subtle, but I think you are trying to make things operate in a more secure manner? this makes this logic always return a valid target if there is one, right?

Reply:
yes, I am gonna improve here and yield some warning/error messages.